src/bundle/Resources/views/themes/admin/account/profile/view.html.twig

```diff
@@ -58,22 +58,25 @@
                     {% set items = [
                         {
                             label: 'user_profile.summary.full_name.label'|trans|desc("Full name"),
-                            content: ibexa_content_name(content)
+                            content: ibexa_content_name(content),
                         },
                         {
                             label: 'user_profile.summary.email.label'|trans|desc("Email"),
-                            content_raw: email_link
+                            content_raw: email_link,
                         }
                     ] %}
 
                     {% for group in field_groups %}
                         {% set field_definitions = content.getContentType().getFieldDefinitions().filterByGroup(group) %}
-
+                        
```

Comment:
```suggestion

```

Reply:
It is set before if. Why I should remove newline?

Reply:
Or maybe u mean newline instead of tab space?